frontend/src/app/globals.css

```
@@ -31,3 +31,8 @@
     transform: translateY(0) scale(1);
   }
 }
+
+/* タイトルのフェードインアニメーション(link修正)*/
```

Stylelint (17.10.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)